docs/deployment.md

```diff
@@ -38,7 +38,8 @@ You can get the Chroma Docker image from [Docker Hub](https://hub.docker.com/r/c
 
 ```sh
 docker pull chromadb/chroma
-docker run -p 8000:8000 chromadb/chroma
+docker volume create chroma-data
```

Comment:
@jarqvi, in your experience, why do you think this is better than local dir binding (`-v ./chroma-data:/chroma/chroma`)? Is it better for developer experience to create a separate volume?

Reply:
I usually use volumes to store container data and bind mounts to configure or dynamically change the container.